docs/samples/Microsoft.ML.Samples/Dynamic/MatrixFactorization.cs

```diff
@@ -48,8 +39,8 @@ public static void MatrixFactorizationInMemoryData()
         {
             // Create an in-memory matrix as a list of tuples (column index, row index, value).
             var dataMatrix = new List<MatrixElement>();
-            for (uint i = _synthesizedMatrixFirstColumnIndex; i < _synthesizedMatrixFirstColumnIndex + _synthesizedMatrixColumnCount; ++i)
-                for (uint j = _synthesizedMatrixFirstRowIndex; j < _synthesizedMatrixFirstRowIndex + _synthesizedMatrixRowCount; ++j)
+            for (uint i = 0; i < _synthesizedMatrixColumnCount; ++i)
```

Comment:
This change doesn't look correct. This example aims at testing user data set where indexes starting from `_synthesizedMatrixFirstColumnIndex=1`.
```suggestion
            for (uint i = 0; i < _synthesizedMatrixColumnCount; ++i)
```